### DISCUSSION TITLE
Considering using grimblast as screenshot tool

### DISCUSSION THREAD

**Discussion by clsty** [end-4/dots-hyprland#290] [Ideas]
I just found [grimblast](https://github.com/hyprwm/contrib/tree/main/grimblast).

It supports some cool features, e.g. freeze when picking area, auto select window when hovering.

See its doc [here](https://github.com/hyprwm/contrib/blob/main/grimblast/grimblast.1.scd), which also provides a config example for Hyprland.

And it's available in AUR.

P.S. It's also first time I saw [hyprwm/contrib](https://github.com/hyprwm/contrib). Very interesting repo.

**Comment by end-4** [comment]
i know this but i still wanted tinted slurp so didn't use it lol
but now, sure
i think i'll just redistribute it since the aur package lists hyprland as a dependency which i don't like

**Reply by end-4** [reply]
okay line 229 of grimblast
turns out we can have SLURP_ARGS env var for color tint, nice
